languages/sardinian/apk/src/main/play/listings/it-IT/short-description.txt

```diff
@@ -0,0 +1 @@
+Pacchetto per la lingua sarda per AnySoftKeyboard
```

Comment:
Why is this under `it-IT`? Shouldn't it be under `sc` locale?

Reply:
There is not a Sardinian version of the Play Store yet and a lot of Sardinians use it in Italian, for now. So I added Italian to be make it more easy to understand for them. Anyway, I added Sardinian (as "sc-IT", or should I change it to sc?) as well, just in case they will add it soon.